docs/gathered-data.md

```diff
@@ -180,9 +181,9 @@ Output raw size: 108
 #### MostRecentMetrics
 [{"Name":"config/metrics","Captured":"0001-01-01T00:00:00Z","Fingerprint":"","Item":"SGVsbG8sIGNsaWVudAo="}]
 
-## UnhealthyNodes
+## Nodes
 
-collects all unhealthy Nodes.
+collects all Nodes.
```

Comment:
There is still the `unhealthy` word referenced in the the next paragraph, that seems to not be relevant anymore